ClientApp/viking-vault/src/display_user_list/AdminPage.tsx

```diff
@@ -106,7 +106,7 @@ class AdminPage extends React.Component<any, IProfileData>{
                 <TopBar/>
                 <UserIcon></UserIcon>  
```

Comment:
If `<UserIcon>` component does not receive children as in: `<tag> children </tag>`, consider replacing with self closing tag `<UserIcon />`